index.html

```diff
@@ -4,9 +4,226 @@
+                    </li>
+                    <li class="nav-item me-5">
+                        <a class="nav-link black-color" href="#">Join</a>
+                    </li>
+                    <li class="nav-item me-5">
+                        <a class="nav-link black-color" href="#">News</a>
+                        </li>
+                    
+                    </ul>
+            </div>
+            <div class="right-side">
+                <button class="campaign-btn"> Order Now</button>
+            </div>
+        </div>
+    </nav>
+      
+    <section class="header">
+        <div class="container h-100 ">
+            <div class="content h-100 d-flex  flex-column justify-content-center">
+                <div class="pragrahph-one">
+                    <p>"Hello! Sharing Dishes!"</p>
+                </div>
+                <div class="heading">
+                    <h1>EXPERIENCE THE BEST 4 CHEFS<br> NEW BRANCH IN CAIRO <br>2023</h1>
+                </div>
+                <div class="pragrahph-two">
+                    <p>Indulge in a special tasting menu, meet and greet with the chefs, witness a cooking competition, and enjoy a wine pairing dinner featuring dishes created by the best 4 chefs</p>
+                </div>
+                <div class="footer-header mt-5 black-color">
+                    <h2>2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p>@ National Museum of Egypt, Doki cairo</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    </header>
+    <section class="dishes py-5 mb-5 black-background white-color">
+        <div class="container">
+            <div class="title text-center d-flex justify-content-center">
+                <h2>Main Program</h2>
+            </div>
+            <div class="data row justify-content-center mt-5">
+                <div class="col-md d-flex flex-sm-column flex-md-column align-items-center opacity">
+                    <div class="icon mb-2 me-2">
+                        <i class="fa-solid fa-utensils"></i>
+                    </div>
+                    <div class="title mb-2 me-2 orange-color">
+                        Chef's Tasting Menu
+                    </div>
+                    <div class="description mb-2">
+                        Experience a special tasting menu featuring signature dishes from each of our 4 top chefs. Each dish will be introduced by the chef who created it, giving you a chance to learn more about their culinary inspirations and techniques.
+                    </div>
+                </div>
+                <div class="col-md d-flex flex-sm-column flex-md-column align-items-center opacity">
+                    <div class="icon mb-2 me-2">
+                        <i class="fa-solid fa-people-group"></i>
+                    </div>
+                    <div class="title mb-2 me-2 orange-color">
+                        Meet the Chefs
```

Comment:
- [ ] Please fix the spacing issue in your `main program` section if you check your `main program mobile version` section there are lots of spacing issues so kindly fix this issue :+1: check below :point_down: the comparisons between your design and Bahnace design :slightly_smiling_face: 

**Your Design**
![screencapture-127-0-0-1-5500-index-html-2023-04-20-19_47_35 (1)](https://user-images.githubusercontent.com/45226848/233412695-6f675834-0204-432d-ae65-0fe6bc61551b.png)

**Bahance Design**
![Screenshot from 2023-04-20 19-50-10](https://user-images.githubusercontent.com/45226848/233412784-7bcb66ad-4c64-44fd-a71d-c5303150e257.png)